includes/class-notices.php

```diff
@@ -409,33 +408,6 @@ function (response) {
 		endif;
 	}
 
-	/**
-	 * Show notices
-	 * Note: only for internal use
-	 *
-	 * @since 2.3.0
-	 */
-	public function __reveal_notices() {
-		?>
-		<script>
-			jQuery(document).ready(function($){
-				// Fix notice appearance issue.
-				window.setTimeout(
-					function(){
-						var give_notices = $('.give-notice');
-
-						if( give_notices.length ) {
-							give_notices.slideDown();
```

Comment:
Update: In testing locally, it works to simply call `.show()` on `ready`. This prevents the "flicker" effect

Reply:
I think `.show()` looks better, but it happens noticeably later. While this would reduce mis-clicks, I think it is a little too slow.

The flicker doesn't look great, but it happens almost as fast as the page is rendered, which prevents mis-clicks.

Reply:
Did you try `show()` on `ready`, without using the timeout at all? In my testing it seems to appear about as quickly as the flicker occurs. I assume the flicker is also the result of an `onready` call happening somewhere else. 

But, its also possible I'm wrong in that! If you're experiencing `show()` as noticeably slower even when skipping the timeout, we should probably stick with what you have in this PR.

Reply:
@henryholtgeerts Ok! Yes, I tried `.show()` without the timeout - mucho gusto. PR updated.